CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.9.2)
```

Comment:
Just curious, why was the CMake's version demoted?

Reply:
3.9.2 is the highest version supported by Travis.